pom.xml

```diff
@@ -88,7 +88,7 @@
         <dependency>
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-validator</artifactId>
-            <version>5.2.2.Final</version>
+            <version>5.3.0.Alpha1</version>
```

Comment:
@gmandnepr sorry, I've just read description of PR. I think it's fine to use alpha provided all tests are passing